### PR TITLE
Sepia

### PR DESCRIPTION
**Pull request by ghost**
Adds a new sepia slime reaction, timestop. Activated with plasma, it
creates a bubble that freezes any mobs/projectiles in place for the
duration. The creator of the bubble is immune to its effects. Currently a 5x5 area

When it ends, projectiles continue on their original path.

Moved camera and film to blood and water, respectively, if anyone even
cared about those.

Might need tweaking/balancing, but considering scientists can make bombs
and this takes longer, I don't think it'll be that bad (especially since
you can't melee people in the field)

### CODE REVIEW

**Comment by Iamgoofball** [comment]
>thumbs.db

File: code/modules/reagents/Chemistry-Recipes/Slime_extracts.dm [hunk 1]
```diff
@@ -559,22 +577,4 @@
 /datum/chemical_reaction/slimefilm/on_reaction(var/datum/reagents/holder)
 	feedback_add_details("slime_cores_used","[replacetext(name," ","_")]")
 	var/obj/item/device/camera_film/P = new /obj/item/device/camera_film
-	P.loc = get_turf(holder.my_atom)
-
-//Pyrite
-/datum/chemical_reaction/slimepaint
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Recipes/Slime_extracts.dm]: why was pyrite removed?

File: tgstation.dme [hunk 2]
```diff
@@ -6,6 +6,52 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "html"
+#define FILE_DIR "icons"
+#define FILE_DIR "icons/ass"
+#define FILE_DIR "icons/effects"
```
Comment (Iamgoofball) [inline tgstation.dme]: gotta disable this in the DM options menu

**Comment by Iamgoofball** [comment]
>deleted

RIP

File: code/modules/projectiles/projectile.dm [hunk 3]
```diff
@@ -130,17 +130,19 @@
...
 				return
-			kill_count--
-			if((!( current ) || loc == current))
-				current = locate(Clamp(x+xo,1,world.maxx),Clamp(y+yo,1,world.maxy),z)
-			step_towards(src, current)
-			if((original && original.layer>=2.75) || ismob(original))
-				if(loc == get_turf(original))
-					if(!(original in permutated))
-						Bump(original, 1)
+			if(!paused)
```
Comment (optimumtact) [inline code/modules/projectiles/projectile.dm]: Wouldn't it be better to remove the bullet from the processing list and then readd it at the end of the timewarp?

Comment (RemieRichards) [inline code/modules/projectiles/projectile.dm]: This isn't process() @optimumtact, Aran changed it to be a spawn()'d off While loop ages ago.

**Pull request closed without merging**